packages/website/src/components/Benefits/Benefits.tsx

```diff
@@ -0,0 +1,81 @@
+import { useState } from 'react';
+
+import { BenefitCard, BenefitCardProps } from '@coderscamp/ui/components/BenefitCard';
+import { Button } from '@coderscamp/ui/components/Button';
+import { Center } from '@coderscamp/ui/components/Center';
+import { Grid } from '@coderscamp/ui/components/Grid';
+import { Typography } from '@coderscamp/ui/components/Typography';
+import { useBreakpointValue } from '@coderscamp/ui/hooks/useBreakpointValue';
+import { SolidArrowDownIcon, SolidArrowUpIcon } from '@coderscamp/ui/icons';
+
+export interface BenefitsProps {
+  title: string;
+  benefitItems: BenefitCardProps[];
+  showExpandProfitButton?: boolean;
+}
+
+const expandedProfitButton = {
+  props: {
+    position: 'sticky',
+    bottom: '20px',
+    icon: <SolidArrowUpIcon />,
+  },
+  text: 'Zwiń listę zalet CodersCamp',
+} as const;
+const rolledUpProfitButton = {
+  props: {
+    icon: <SolidArrowDownIcon />,
+  },
+  text: 'Rozwiń listę zalet CodersCamp',
+} as const;
+
+const BENEFITS_AFTER_SHRINKAGE = 6;
```

Comment:
💭 It could be great to pass something like `shirinkSize` as a required prop instead of `showExpandProfitButton`. This way we can customize the number of displayed benefits per component usage. It would be useful when we for example decide that we want to sharing mentor benefits count but for a different number of tiles.

Reply:
I changed that, with one exception, added shirinkSize as optional and every elements will be displayed if it won't be given